Create solucion_bernardo.py

zelayab opened the pull request:
Cree la solución para el ejercicio de práctica.
Si es necesario corregir por favor avíseme profe.

zelayab committed:
Create solucion_bernardo.py

zelayab committed:
Update solucion_bernardo.py

enzzo19 commented:
Hola Bernardo probe la funcion y devuelve algo levemente diferente mira, la consigna dice que la funcion al recibir “Hola, tú“ debe retornar “H o l a , t ú “ y tu funcion retorna “H o l a ,  t ú “ osea que tiene un espacio mas entre la "," y la siguiente palabra. Controla esooo y avisame si necesitas ayuda para poder resolverlo.

zelayab committed:
Update solucion_bernardo.py

Añadi la corrección al error que generaba más espaciado después de la coma y queda bien.

zelayab commented:
Hola profe, ahi volví a actualizar, le quite el espaciado de más que tenía después de la coma y probé de ver como salía el output y lo copie y lo pegue y lo puse dentro del código como comentario.
Cualquier cosa si hay que corregirlo, lo hago !
